pyairtable/api/table.py

```diff
@@ -16,7 +16,15 @@ class Table(ApiAbstract):
     base_id: str
     table_name: str
 
-    def __init__(self, api_key: str, base_id: str, table_name: str, *, timeout=None):
+    def __init__(
+        self,
+        api_key: str,
+        base_id: str,
+        table_name: str,
+        *,
+        timeout=None,
+        max_records_per_request=DEFAULT_MAX_RECORDS_PER_REQUEST,
+    ):
```

Comment:
See my reasoning above for `Api`

```suggestion
    def __init__(self, api_key: str, base_id: str, table_name: str, **kwargs: Any):
```
_NB, don't forget to add the import for `Any` from `typing` above_